client/client.js

```diff
@@ -0,0 +1,15 @@
+$(function () {
+var socket = io();
+var bruger = "Bob";
+//Math.random();
```

Comment:
Seems like some dead codes sneaked in here (`//Math.random()`), let us remove that. : )